pages/common/diffstat.md

```diff
@@ -0,0 +1,11 @@
+# diffstat
+
+> Make histogram of diff-output.
+
+- Display changes in a histogram:
+
+`diff {{file1}} {{file2}} | diffstat`
+
+- Display inserted, deleted, modified changes in table format:
```

Comment:
I think you need an `and` after the last comma in that list.

Perhaps `as a table` instead of `in table format` would read better?